Source/Scene/PointCloudPostProcessor.js

```diff
@@ -72,20 +78,29 @@ define([
         var screenWidth = context.drawingBufferWidth;
         var screenHeight = context.drawingBufferHeight;
 
-
         var colorTextures = new Array(2);
-        var depthTextures = new Array(2);
-        var framebuffers = new Array(2);
+        var depthTextures = new Array(3);
+
+        var ecTexture = new Texture({
+            context : context,
+            width : screenWidth,
+            height : screenHeight,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : PixelDatatype.FLOAT,
```

Comment:
Do we need to check for an extension for this?

WebGL 2 or https://www.khronos.org/registry/webgl/extensions/OES_texture_float/